gendocs/gen_cli_api_reference.sh

```diff
@@ -26,6 +38,28 @@ gen_cmd_docs 'disable-telemetry'
 gen_cmd_docs 'list-packages'
 gen_cmd_docs 'push'
 
+# Document environment variables
+echo '## Environment variables' >> cli.md
+gen_env_docs QUILT_DISABLE_USAGE_METRICS '$ export QUILT_DISABLE_USAGE_METRICS=true' \
+'Disable anonymous usage collection. Defaults to `False`'
+
+gen_env_docs QUILT_MINIMIZE_STDOUT '$ export QUILT_MINIMIZE_STDOUT=true' \
+'Turn off TQDM progress bars for log files. Defaults to `False`'
+
+
+# Document Constants
+echo '## Constants (see [util.py](https://github.com/quiltdata/quilt/blob/master/api/python/quilt3/util.py) for more)' >> cli.md
+echo '- `APP_NAME` - Name of the application' >> cli.md
+echo '- `APP_AUTHOR` - Author' >> cli.md
+echo '- `BASE_DIR` - Base directory of the application' >> cli.md
+echo '- `BASE_PATH` - Base path(pathlib) of the application directory' >> cli.md
+echo '- `CACHE_PATH` - User cache directory ' >> cli.md
+echo '- `TEMPFILE_DIR_PATH` - Tempfile directory' >> cli.md
```

Comment:
These comments should either be useful :) or delete them. You might have to read the code a bit do see what they do.